game.project

```diff
@@ -1,10 +1,11 @@
 [project]
 title = Platypus
 version = 0
-dependencies = https://github.com/britzl/defold-orthographic/archive/2.2.5.zip,https://github.com/britzl/defold-input/archive/2.8.3.zip
+dependencies#0 = https://github.com/britzl/defold-orthographic/archive/master.zip
+dependencies#1 = https://github.com/britzl/defold-input/archive/2.8.3.zip
 
 [bootstrap]
-main_collection = /example/grottoescape/grottoescape.collectionc
+main_collection = /example/variable_raycast_collisions/varraycasts.collectionc
```

Comment:
Wasn't the idea to use the starter.collection?

Reply:
For quicker tests i have put directly the  varraycast collection as bootstrap (avoid a click each time).
I've forgotten to restore starter collection before PR.
And yes, there was a problem with the buttons. I thought it was random, because i saw it two or three times. Usually i click in the middle of the buttons, so i was thinking it was a random bug. I wanted to talk about this on the forum, but i've forgotten :/